hunt/db/migrate/20200731184546_create_animals.rb

```diff
@@ -4,6 +4,9 @@ def change
       t.string :name
       t.string :category
       t.string :state
+      t.text :description
+      t.date :season_open
+      t.date :season_close
```

Comment:
Typically we wont want to add to existing migration files, because if this app was already deployed somewhere, we would have to reset the database in order to pick up these changes. Since this is just local and we have the importer this is fine for now 👍 

Just for reference, the way to do that is to generate a new migration file (`rails g migration AddColsToAnimals`) and then you could add columns to the existing table like this: 
```
class AddColsToAnimals < ActiveRecord::Migration[6.0]
  def change
    add_column :animals, :season_open, :date
    add_column :animals, :season_close, :date
    add_column :animals, :description, :text
  end
end
```